app/posts/register-trainee-teachers/2021-08-03-highest-academic-qualification.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Removing the highest academic qualification question
+description: Why we don’t think we need this question and an alternate design if we do.
+date: 2021-08-03
+---
+{% from "figure/macro.njk" import appFigure with context %}
+
+For trainees on undergraduate courses the provider has been asked for the trainee’s highest academic qualification.
```

Comment:
Annoyingly I don't think we did a blog post on highest qualification. Without that, can we add a screenshot for the current page here?

Reply:
👍  will do